packages/@uppy/dashboard/src/style.scss

```diff
@@ -1,12 +1,13 @@
-@import '@uppy/core/src/_utils.scss';
-@import '@uppy/core/src/_variables.scss';
-@import '@uppy/informer/src/style.scss';
-@import '@uppy/status-bar/src/style.scss';
-@import '@uppy/provider-views/src/style.scss';
+@use "sass:color";
+@use '../../core/src/_utils.scss';
+@use '../../core/src/_variables.scss';
+@use '../../informer/src/style.scss';
+@use '../../status-bar/src/style.scss' as style2;
+@use '../../provider-views/src/style.scss' as style3;
```

Comment:
what's the purpose of these and why are they not named better? i can't see style2 and style3 being referenced anywhere